### PR TITLE
Closes #360 | Implement dataloader for khpos

### LINKED ISSUE #360: Create dataset loader for khPOS
Dataloader name: `khpos/khpos.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?khpos

|     Dataset| khpos |
|-------------|---|
| Description | khPOS is a POS-tagging dataset for the Khmer language. The data was collected from websites, student and voter lists. It was automatically tokenized with a CRF model and then manually tagged for POS and checked for errors. |
| Subsets | - |
| Languages | khm |
| Tasks | POS Tagging |
| License | Creative Commons Attribution Non Commercial Share Alike 4.0 (cc-by-nc-sa-4.0) |
| Homepage | https://github.com/ye-kyaw-thu/khPOS/tree/master |
| HF URL | https://huggingface.co/datasets/seanghay/khPOS |
| Paper URL | https://github.com/ye-kyaw-thu/khPOS/blob/master/khpos.pdf |

### PR DESCRIPTION
**Pull request by luckysusanto**
Closes #360 

### Checkbox
- [X] Confirm that this PR is linked to the dataset issue.
- [X] Create the dataloader script `seacrowd/sea_datasets/khpos/khpos.py` (please use only lowercase and underscore for dataset naming).
- [X] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [X] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [X] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [X] Confirm dataloader script works with `datasets.load_dataset` function.
- [X] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/khpos/khpos.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Review by SamuelCahyawijaya — CHANGES_REQUESTED** [review]
Hi @luckysusanto , thank you for contributing. The code works perfectly, nonetheless, according to the original dataset, there are also validation and test splits. 

I checked the original repo and I found these two files:
Validation Set: https://github.com/ye-kyaw-thu/khPOS/blob/master/corpus-draft-ver-1.0/data/OPEN-TEST
Test Split: https://github.com/ye-kyaw-thu/khPOS/blob/master/corpus-draft-ver-1.0/data/CLOSE-TEST

Would you be able to implement them in the dataloader as well? I think it simply parse the word and tag pair based on "/". I think it will be very similar to the [implementation on HF](
https://huggingface.co/datasets/seanghay/khPOS/blob/006b410cf0a17b77b448fe3d20d11ae6a8338951/khPOS.py#L50-L69). 

Thank you and do let me know if you have any problem with it!

**Comment by luckysusanto** [comment]
Hello @SamuelCahyawijaya, my bad for not noticing the additional validation and test set.

I've added them to the dataloader! Requesting re-review :)

**Review by SamuelCahyawijaya — APPROVED** [review]
@luckysusanto : Thanks for the update! LGTM!

File: seacrowd/sea_datasets/khpos/khpos.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+"""
+The khPOS Corpus (Khmer POS Corpus) is a 12,000 sentences (25,626 words) manually word segmented and POS tagged corpus 
+developed for Khmer language NLP research and developments. We collected Khmer sentences from websites that include
+various area such as economics, news, politics. Moreover it is also contained some student list and voter list of 
+national election committee of Cambodia. The average number of words per sentence in the whole corpus is 10.75. 
+Here, some symbols such as "។" (Khmer sign Khan), "៖" (Khmer sign Camnuc pii kuuh), "-", "?", "[", "]" etc. also 
+counted as words. The shortest sentence contained only 1 word and longest sentence contained 169 words. This dataset contains
+A validation set and a test set, each containing 1000 sentences.
+"""
+import os
```
Comment (ryanignatius) [inline seacrowd/sea_datasets/khpos/khpos.py:25]: please remove unused import

File: seacrowd/sea_datasets/khpos/khpos.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value 
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://raw.githubusercontent.com/ye-kyaw-thu/khPOS/master/corpus-draft-ver-1.0/data/after-replace/train.all2",
+    'validation': "https://raw.githubusercontent.com/ye-kyaw-thu/khPOS/master/corpus-draft-ver-1.0/data/OPEN-TEST",
+    'test'      : "https://raw.githubusercontent.com/ye-kyaw-thu/khPOS/master/corpus-draft-ver-1.0/data/CLOSE-TEST"
+}
```
Comment (ryanignatius) [inline seacrowd/sea_datasets/khpos/khpos.py:70]: I think it's better to structure it like this,
```
_URLS = {
    _DATASETNAME: {
        "train": ...,
        "validation": ...,
        "test": ...,
    },
}
```
wdyt?

**Review by ryanignatius — COMMENTED** [review]
reviewed and tested

File: seacrowd/sea_datasets/khpos/khpos.py (hunk 2 above)
Comment (luckysusanto) [reply seacrowd/sea_datasets/khpos/khpos.py]: Yeah, this is definitely better! Changed!

File: seacrowd/sea_datasets/khpos/khpos.py (hunk 1 above)
Comment (luckysusanto) [reply seacrowd/sea_datasets/khpos/khpos.py]: Gotcha, thanks for that input

**Comment by luckysusanto** [comment]
Requesting re-review @ryanignatius!

**Review by ryanignatius — APPROVED** [review]
